client/python/kserve-api/samples/README.md

```diff
@@ -258,6 +258,13 @@ optional arguments:
                         Define pipeline name, must be same as is in service
   --dag-batch-size-auto
                         Add demultiplexer dimension at front
+  --tls                 use TLS communication with gRPC endpoint
+  --server_cert SERVER_CERT
+                        Path to server certificate
+  --client_cert CLIENT_CERT
+                        Path to client certificate
+  --client_key CLIENT_KEY
+                        Path to client key
```

Comment:
I wonder if we want to have 4 arguments (4th being ssl) when if you use one of (client|server)_(cert|key) you know that client want to use ssl?